tests/integration/conftest.py

```diff
@@ -88,11 +88,11 @@ async def gh_app_fixture(
         "github-actions-exporter-image": pytestconfig.getoption("--githubactionsexporter-image"),
     }
     charm = await ops_test.build_charm(".")
-    application = await ops_test.model.deploy(
-        charm, resources=resources, application_name=app_name, series="focal"
-    )
-
-    await dependencies
+    async with ops_test.fast_forward():
```

Comment:
Out of curiosity, what is this for?

Reply:
This is to speed up update-status temporarily ;-)
https://juju.is/docs/sdk/integration-testing-cookbook